_quarto.yml

```diff
@@ -39,7 +39,6 @@ website:
               href: community/CONTRIBUTING.md
             - auto: community/guide-entries
             - auto: community/decisions
```

Comment:
```suggestion
            - auto: community/decisions
            - text: "Code of Conduct"
              href: "https://github.com/seedcase-project/.github/blob/main/CODE_OF_CONDUCT.md"
```

Reply:
We still would like to link to the Code of Conduct, so I've added it as an href.